rholang/src/main/scala/coop/rchain/rholang/interpreter/storage/implicits.scala

```diff
@@ -45,13 +45,9 @@ object implicits {
                   case (freeMap: FreeMap, caughtRem: Seq[Channel]) =>
                     val remainderMap = pattern.remainder match {
```

Comment:
I think the two cases above can also be merged in a single pattern match - even if used in `map`